flepimop/gempyor_pkg/src/gempyor/config_validator.py

```diff
@@ -130,7 +142,10 @@ def which_value(cls, values):
         value = values.get('value') is not None
         timeseries = values.get('timeseries') is not None
         if value and timeseries:
-            raise ValueError('Error in seir::parameters: your parameter is both a timeseries and a value, please choose one')
+            raise ValueError(
+                f'Configuration error in seir::parameters: your parameter is both a timeseries and a value, please choose one. '
+                f'Current values - value: {values.get('value')!r}, timeseries: {values.get('timeseries')!r}.'
+            )
```

Comment:
not sure where we're tracking this, but: this also seems like a configuration schema conceptual error. if options are mutually exclusive, it would ideally impossible to specify both (mild caveat that we want to avoid too much overloading of keys)